src/test.ts

```diff
@@ -15,6 +15,6 @@ getTestBed().initTestEnvironment(
   platformBrowserDynamicTesting()
 );
 // Then we find all the tests.
-const context = require.context('./', true, /\.spec\.ts$/);
+// const context = require.context('./', true, /\.spec\.ts$/);
 // And load the modules.
-context.keys().map(context);
+// context.keys().map(context);
```

Comment:
@whyboris why this is commented?

Reply:
I believe this code is no longer needed. If I remember right, when you create a new Angular application with the CLI this code is not present 🤔 

Similar code change in a repository that seems to be up to date:
https://github.com/maximegris/angular-electron/commit/680d18df3776617ccaf61bf5f19be5189d938259